roles/dtc/common/templates/ndfc_vrf_lite.j2

```diff
@@ -1,6 +1,6 @@
 {% if item.ospf is defined and item.ospf %}
 {% include 'ndfc_vrf_lite_ospf.j2' %}
```

Comment:
To keep this file consistent with spacing, we should also add 2 spaces on line 2.

Reply:
thanks @juburnet ! spaces added